apps/modernization-ui/src/components/EditableCard/EditableCard.tsx

```diff
@@ -12,15 +12,16 @@ export type Props = {
     editing?: boolean;
     onEdit: () => void;
     children: ReactNode;
+    disableEdit?: boolean;
```

Comment:
It's good to keep boolean variables affirmative, can `disableEdit` be renamed?  Maybe something like `editable` but then you have to flip the value when passed to `disabled`.  

Does `readOnly` make sense?  it retains the same boolean value as `disableEdit` and is affirmative.

Reply:
I like the idea of renaming while keeping it affirmative. Nice thoughts , thanks Adam!